stories/core/Table.stories.tsx

```diff
@@ -307,6 +308,98 @@ Sortable.args = {
   isSortable: true,
 };
 
+export const Expandable: Story<TableProps> = (args) => {
+  const { columns, data, ...rest } = args;
+
+  const onExpand = useCallback(
+    (state) => action(`Expanded. Table state: ${JSON.stringify(state)}`)(),
+    [],
+  );
+
+  const tableColumns = useMemo(
+    () => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'expander',
```

Comment:
Oh, I think we want to provide this from us. Something like selectable.
Cause this is defined by standards, so we could control that from us, if expanded is enabled (need new prop?). And then check, if we can give users ability to override this if they want.

Reply:
I moved the expander column to table with latest commit. The behavior can be disabled, if "provideDefaultExpander" prop (true by default) is set to false.

@veekeys What do you think about this?

Reply:
I do not think expand row should be true by default.. Users should turn it on, when they need it.
I like the hook (same as select). Now we need to think, how we could allow this for the user to customize.. I guess same is applicable to selection column.

Reply:
For me at least, customizing expander from the user side is pretty easy, I only need to add additional Cell at the beggining which invokes `props.row.toggleRowExpanded()` on button click.

I made `provideDefaultExpander` to be `false` by default.

Reply:
I think we could provide object/map of predefined columns. It could have expander and maybe even selection. I now think we shouldn't have provided that `isSelectable` flag...

Reply:
Maybe instead of `provideDefaultExpander`, we should have a prop for function like `canExpand` or `isExpandable` which would return a boolean whether row should have expansion button.